rosbag2_transport/include/rosbag2_transport/rosbag2_transport.hpp

```diff
@@ -74,7 +74,9 @@ class Rosbag2Transport
    * \param play_options Options regarding the playback (e.g. queue size)
    */
   ROSBAG2_TRANSPORT_PUBLIC
-  void play(const StorageOptions & storage_options, const PlayOptions & play_options);
+  void play(
+    const StorageOptions & storage_options, const PlayOptions & play_options,
+    const std::vector<std::string> & topic_remapping_options = {});
```

Comment:
Do you think it would be more appropriate to add `topic_name_map` to `PlayOptions`/`RecordOptions` rather than adding new arguments to the API?